### ISSUE TITLE
Kaldi feedstock update on conda-forge

### ISSUE THREAD

**Issue by mmcauliffe** [kaldi-asr/kaldi#4661]
Hello,

I've recently updated the [kaldi feedstock for conda-forge](https://github.com/conda-forge/kaldi-feedstock), and I was wondering if any Kaldi maintainers were interested in being added a maintainer there as well. 

To get it working with the conda-forge system, I made some [changes to the CMake build system](https://github.com/conda-forge/kaldi-feedstock/blob/master/recipe/cmake.patch).  To compile some notes on these:

* Kaldi on conda-forge doesn't link to `libfst.so`, but rather to `libfst.16.so` (to allow for installation beside the OpenFst conda-forge package which is 1.8.1)
* On Windows, OpenFst binaries are included as well as Kaldi binaries (OpenFst on Conda doesn't support Windows)
* Removed building of `online` and `online2` on Windows, as I was getting some compiler errors
* Added support for building `fgmmbin` on all platforms

I can make a PR for the ones that would be helpful generally (i.e. `fgmmbin` included in the CMake build).

**Comment by jtrmal** [comment]
Yes, would be good if you create a PR... some of these things should be useful.
Thanks!

**Comment by mmcauliffe** [comment]
@jtrmal PR created here: #4669
